index.js

```diff
@@ -53,7 +53,9 @@ exports.getWindowBoundsCentered = options => {
 
 	const [width, height] = options.window.getSize();
 	const windowSize = options.size || {width, height};
-	const screenSize = api.screen.getDisplayNearestPoint(api.screen.getCursorScreenPoint()).workArea;
+	const screenSize = options.includeFullDisplayBounds ?
+		api.screen.getDisplayNearestPoint(api.screen.getCursorScreenPoint()).bounds :
+		api.screen.getDisplayNearestPoint(api.screen.getCursorScreenPoint()).workArea;
```

Comment:
There's a lot of duplication here. Extract `api.screen.getDisplayNearestPoint(api.screen.getCursorScreenPoint())` to a variable.